modules/ocf/manifests/packages/ntp.pp

```diff
@@ -8,6 +8,17 @@
       content => template('ocf/ntp.conf.erb'),
       require => Package['ntp'],
     }
+
+    #firewall input rule, allow ntp (123 udp/tcp)
+    ocf::firewall::firewall46 {
+      '101 accept all ntp':
+        opts => {
+          chain  => 'PUPPET-INPUT',
+          proto  => ['tcp', 'udp'],
```

Comment:
Unfortunately you can't pass an array into here, you need to add these rules individually or by using `each`.

Ditto for every other place this pattern occurs.

Reply:
imo this is something our firewall type should do for us

Reply:
Hence my comment about [firewall_multi](https://forge.puppet.com/alexharvey/firewall_multi). I tried using that module, but unfortunately it doesn't forward dependencies, so it's not usable inside pre.pp or post.pp.

Reply:
Resolved